src/Stays/Stays.spec.ts

```diff
@@ -0,0 +1,36 @@
+import nock from 'nock'
+import { Duffel } from '../index'
+import { MOCK_SEARCH_RESULT } from './mocks'
+
+const duffel = new Duffel({ token: 'mockToken' })
+describe('Stays', () => {
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  it('should post to /stays/search when `search` is called', async () => {
+    const mockResponse = { data: { results: [MOCK_SEARCH_RESULT] } }
+    const mockSearchParams = {
+      location: {
+        radius: 5,
+        geographic_coordinates: {
+          latitude: 40.73061,
+          longitude: -73.935242,
+        },
+      },
+      check_in_date: '2023-10-20',
+      check_out_date: '2023-10-24',
+      adults: 2,
+      rooms: 1,
+    }
+
+    nock(/(.*)/)
```

Comment:
Possibly a dumb question, but what's the benefit of these tests where we write some data, make a mock API response using the data, then make a request to just receive the data we wrote?

Reply:
Not a dumb question! 

The important part of these tests are simply about ensuring that we are passing the right data to the corresponding endpoint. The assertion of the response is just kinda there to demonstrate that these endpoints simply pass the responses through (rather than doing any sort of data transformation). 

One could argue that that's not very useful (given that we never perform any transformation on any response in this SDK), but it probably doesn't hurt to have them either I think.